classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -32,6 +49,134 @@ class WebGatewayProvider implements BootableProviderInterface, ServiceProviderIn
 {
     public function register(Container $app)
     {
+        // Controllers
+        $app[DashboardController::class] = function ($app) {
```

Comment:
This function is becoming extremely large now.

Maybe we can extract the creation of the controllers to another provider.

Reply:
If you migrate to Symfony, that class will be gone soon, anyway. I wouldn't put too much effort in splitting it up right now.

Reply:
@derrabus There is no "if", we are definitely moving to Symfony 4!